requirements-sdp.txt

```diff
@@ -7,7 +7,7 @@ chardet==3.0.4
 ci-watson==0.4
 codecov==2.0.15
 coverage==4.5.4
-crds==7.4.1.1
+-e git+https://github.com/spacetelescope/crds@b9a88d6#egg=crds
```

Comment:
Pointing to the (currently unreleased) tag is probably better according to @jaytmiller:
```
-e git+https://github.com/spacetelescope/crds@7.4.1.2#egg=crds
```
Also add in:
```
lxml==4.4.1
Parsley==1.3
```
as those are now required by `crds` to install.